Software/Source/Bursttest.py

```diff
@@ -0,0 +1,264 @@
+
+# Function to count the packet types in the file and return array of count 
+def indbrstpktcnt(macid):
+	global burst_dbg_cnt
+	global burst_acc_cnt
+	global burst_tmp_cnt
+	global burst_bat_cnt
+	global burst_bst_cnt
+	brstpktcnt = [ ]
+	tempfilename = str(macid)+".txt"
+	f2 = open(tempfilename,"r")
+	while True:
+		data = f2.readline()
+		if data:
+			data_obj = json.loads(data)
+			packettype = data_obj["PacketType"]
+			burst_dbg_cnt,burst_acc_cnt,burst_tmp_cnt,burst_bat_cnt,burst_bst_cnt = indpacketcheck(packettype)
+			if packettype == '4':
+				# Create array of data
+				brstpktcnt.append(burst_dbg_cnt)
+				brstpktcnt.append(burst_acc_cnt)
+				brstpktcnt.append(burst_tmp_cnt)
+				brstpktcnt.append(burst_bat_cnt)
+				brstpktcnt.append(burst_bst_cnt)
+                #Clear count
+				burst_dbg_cnt = 0
+				burst_acc_cnt = 0
+				burst_tmp_cnt = 0
+				burst_bat_cnt = 0
+				burst_bst_cnt = 0
+		else:
+			break
+	return brstpktcnt
+
+# Function to Create csv file
+def csvfile(file_mac,Test_time,count1,count):
+	file_name = str(file_mac)+".csv"
+	with open(file_name,"w") as f:
+		write=csv.writer(f,delimiter=',',lineterminator='\n')
+		Test_Time=["Testtime",Test_time]
+		Packet_loss="Pkt loss %"
+		Packet_count="Pkt Cnt"
+		Packet=[' ','Dbg Pkt','Acc Pkt','Temp Pkt','Batt Pkt','Brst Pkt']
+		count1.insert(0,Packet_loss)
+		count.insert(0,Packet_count)
+		# write the data in csv file
+		write.writerow(Test_Time)
+		write.writerow(Packet)
+		write.writerow(count1)
+		write.writerow(count)
+		write.writerow(["Sl.No","Burstwise packet count"])
+
+#Function to write Burst packet data in csv file		
+def createcsvfile(file_mac,Burstpkt):
+	global slno
+	tempfname = str(file_mac)+".csv"
+	with open(tempfname, mode='a') as f:
+		write_file=csv.writer(f,delimiter=',',lineterminator='\n')     
+		slno = slno + 1
+		if Burstpkt: 
+			Burstpkt.insert(0,slno)
+			# write the data in csv file
+			write_file.writerow(Burstpkt)
+		else:
+			slno = 0
+
+# Prompt user to enter test time and burst interval	
+Test_time=input("Enter the TestTime:")
+Burst_interval=float(input("Enter the Burst interval:"))
+
+#Check macid in the file and create array of macid
+comparemacid()
+
+for i in range(len(macid)):
+	print(i)
+	writefile(macid[i])
+	Total_Dbg_Pkt=0
+	Total_Acc_Pkt=0
+	Total_Temp_Pkt=0
+	Total_Batt_Pkt=0
+	Total_Brst_Pkt=0
+	dbg_pkt,acc_pkt,temp_pkt,batt_pkt,brst_pkt=PackeTypeCheck(macid[i])
+	count.append(dbg_pkt)
+	count.append(acc_pkt)
+	count.append(batt_pkt)
+	count.append(batt_pkt)
+	count.append(brst_pkt)
+	print(count)
+	burstinterval1=bursttime(Test_time,Burst_interval)
+	Dbg_Pkt_loss,Acc_Pkt_loss,Temp_Pkt_loss,Batt_Pkt_loss,Brst_Pkt_loss=losscal(burstinterval1,dbg_pkt,acc_pkt,batt_pkt,batt_pkt,brst_pkt)
+	count1.append(Dbg_Pkt_loss)
+	count1.append(Acc_Pkt_loss)
+	count1.append(Temp_Pkt_loss)
+	count1.append(Batt_Pkt_loss)
+	count1.append(Brst_Pkt_loss)
+	print(count1)
+
+index_x=0

```

Comment:
This code will throw error, have you compiled the code before uploading for review...?